src/ResourceManager.cpp

```diff
@@ -0,0 +1,34 @@
+#include "ResourceManager.h"
+
+ResourceManager::ResourceManager() {
+	LoadData();
+}
+
+void ResourceManager::LoadData() {
+	duckyTexture.loadFromFile("Res/JanitorDuck.png");
+}
+
+
+sf::Texture ResourceManager::GetTexture(const char name) {
+	for (auto [key, value] : ResourceManager::TextureMap) {
```

Comment:
nonono :D we have a hash map specificly to acces elements via key
map[name] -> returns value if found
we could add an assert in a non-release version, that will crash the game if a non-existent asset is requested